types/mailgun-js/mailgun-js-tests.ts

```diff
@@ -11,6 +13,23 @@ const mailgun2 = new mailgunFactory2({
     domain: "auth.domain"
 });
 
+const logger = (httpOptions: mailgunFactory.LoggerHttpOptions, payload: string, form: FormData) => {
+    const { method, path } = httpOptions;
+    const hasPayload = !!payload;
```

Comment:
You can remove line 18 to 21. They don't add any value to the tests.

Reply:
removed